omod/src/main/resources/config.xml

```diff
@@ -39,6 +39,18 @@
         <description>The URL of the Frontend 3.0 application files. Only used if `spa.remote.enabled` is true.</description>
     </globalProperty>
 
+	<globalProperty>
+		<property>spa.config.directory</property>
```

Comment:
Why do we need a directory for the config if we're really just managing a single file?